source/includes/fact-viewing-favorite-queries.rst

```diff
@@ -12,4 +12,22 @@
 
    .. step:: Click the :guilabel:`Favorites` button in the past queries pane.
 
-      Select favorite queries.
+      Select your favorited syntax.
```

Comment:
again I don't think syntax is the right word here. You could say "operation" but open to other ideas as long as it's consistent throughout